### PR TITLE
Add yml specifics and force crlf for C# files

### PR DESCRIPTION
**Pull request by mcliment**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -88,3 +90,8 @@ csharp_style_expression_bodied_local_functions = false:suggestion
 csharp_style_expression_bodied_methods = false:silent
 csharp_style_expression_bodied_operators = false:suggestion
 csharp_style_expression_bodied_properties = true:suggestion
+
+[*.yml]
```
Comment (mcliment) [inline .editorconfig:94]: If you try to format yml files, 2 is the standard indentation.

File: .editorconfig [hunk 2]
```diff
@@ -9,6 +9,8 @@ guidelines = 80, 120
 
 # See https://docs.microsoft.com/en-us/visualstudio/ide/editorconfig-code-style-settings-reference
 [*.cs]
+end_of_line = crlf
```
Comment (mcliment) [inline .editorconfig:12]: This is in the recommended link and in general we do like that.

**Review by firas-assaad — APPROVED** [review]

**Comment by mcliment** [comment]
@Grepsy Take a look if you have a minute

**Review by Grepsy — APPROVED** [review]
Nice, agree with all!

**Pull request merged**